src/utils/config-helper.ts

```diff
@@ -39,43 +39,36 @@ export async function fetchConfig (): Promise<any> {
     return Promise.reject(new Error('Could not fetch configuration.json'))
   })
 
-  /**
-   * This is a workaround to fix the sbc-common-components that expect their own session keys.
-   * Ref: #6801
-   */
-  const authApiConfig = {
-    VUE_APP_AUTH_ROOT_API: response.data['SBC_CONFIG_AUTH_API_URL'],
-    VUE_APP_STATUS_ROOT_API: response.data['VUE_APP_STATUS_ROOT_API']
-  }
-  const authConfigString = JSON.stringify(authApiConfig)
-  sessionStorage.setItem('AUTH_API_CONFIG', authConfigString)
-  // console.log('AUTH_API_CONFIG: ' + authConfigString) // don't display
+  const authWebUrl: string = response.data['AUTH_WEB_URL']
+  sessionStorage.setItem('AUTH_WEB_URL', authWebUrl)
+  console.log('Set Auth Web URL to: ' + authWebUrl)
 
   const businessesUrl: string = response.data['BUSINESSES_URL']
   sessionStorage.setItem('BUSINESSES_URL', businessesUrl)
   console.log('Set Businesses URL to: ' + businessesUrl)
 
-  const authUrl: string = response.data['AUTH_URL']
-  sessionStorage.setItem('AUTH_URL', authUrl)
-  console.log('Set Auth URL to: ' + authUrl)
-
   const dashboardUrl: string = response.data['DASHBOARD_URL']
   sessionStorage.setItem('DASHBOARD_URL', dashboardUrl)
   console.log('Set Dashboard URL to: ' + dashboardUrl)
 
-  const legalApiUrl: string = response.data['LEGAL_API_URL']
+  const legalApiUrl: string = response.data['LEGAL_API_URL'] + response.data['LEGAL_API_VERSION'] + '/'
   // set base URL for axios calls
   axios.defaults.baseURL = legalApiUrl
   console.log('Set Legal API URL to: ' + legalApiUrl)
 
-  const authApiUrl: string = response.data['AUTH_API_URL']
+  const authApiUrl: string = response.data['AUTH_API_URL'] + response.data['AUTH_API_VERSION'] + '/'
   sessionStorage.setItem('AUTH_API_URL', authApiUrl)
   console.log('Set Auth API URL to: ' + authApiUrl)
 
-  const payApiUrl: string = response.data['PAY_API_URL']
+  const payApiUrl: string = response.data['PAY_API_URL'] + response.data['PAY_API_VERSION'] + '/'
   sessionStorage.setItem('PAY_API_URL', payApiUrl)
   console.log('Set Pay API URL to: ' + payApiUrl)
 
+  // for system alert banner (sbc-common-components)
+  const statusApiUrl: string = response.data['STATUS_API_URL'] + response.data['STATUS_API_VERSION']
+  sessionStorage.setItem('STATUS_API_URL', statusApiUrl)
```

Comment:
So all apps still need to set the values for components they import (ie, SbcHeader) ? I was hoping the apps wouldn't have to worry about this (ie, the imported components would worry about their own values). Oh well.

Reply:
yes, I think the common components should not have set value logic in it. Only can read the values from project config.